tests/Unit/Activity/ActivityManagerTest.php

```diff
@@ -36,6 +36,7 @@
 use OCP\Share\IShare;
 use PHPUnit\Framework\MockObject\MockObject;
 use Psr\Log\LoggerInterface;
+use Test\TestCase;
 
 use Test\TestCase;
```

Comment:
```suggestion
use Test\TestCase;
```
Somehow this slipped in twice... ;)